.vscode/settings.json

```diff
@@ -4,8 +4,8 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "typescript.enablePromptUseWorkspaceTsdk": true,
   "editor.codeActionsOnSave": {
-    "source.organizeImports.rome": true,
-    "source.fixAll": true
+    "source.organizeImports.rome": "explicit",
```

Comment:
What is causing this, do you know?. I wake up every morning and this change is in my editor.

Reply:
The old keyword has been deprecated - vscode auto updates I should update this in all repos so we don't get it anymore.